docs/website/versioned_docs/version-3.0.0/command-reference/describe-binding.md

```diff
@@ -0,0 +1,48 @@
+---
+title: odo describe binding
+---
+
+`odo describe binding` command is useful for getting information about a service binding. 
+
+This command supports the service bindings added with the command `odo add binding` and bindings added manually to the Devfile, using a `ServiceBinding` resource from one of these apiVersion:
+- `binding.operators.coreos.com/v1alpha1`
+- `servicebinding.io/v1alpha3`
+
+There are 2 ways to describe a service binding:
+- [Describe with access to Devfile](#describe-with-access-to-devfile)
+- [Describe without access to Devfile](#describe-without-access-to-devfile)
+
+## Describe with access to Devfile
```

Comment:
For command reference, we are trying to have a consistent format between all documentation. Can you split these sections up similar to: https://odo.dev/docs/3.0.0/command-reference/dev ?

Ex. Section for Description, Running the Command and Devfile usage.

Reply:
Hi made the changes, thanks